src/main/java/net/dv8tion/jda/bot/sharding/ShardManager.java

```diff
@@ -930,11 +944,11 @@ default VoiceChannel getVoiceChannelById(final String id)
      *
      * <p>This will also change the game for shards that are created in the future.
      *
-     * @param  game
```

Comment:
As far as I know, we always use 2 spaces after `@param` even if it isn't strictly necessary.